avm/res/network/virtual-network-gateway/main.bicep

```diff
@@ -303,7 +312,7 @@ module publicIPAddress 'br/public:avm/res/network/public-ip-address:0.2.0' = [
       publicIpPrefixResourceId: !empty(publicIPPrefixResourceId) ? publicIPPrefixResourceId : ''
       tags: tags
       skuName: skuName == 'Basic' ? 'Basic' : 'Standard'
-      zones: skuName != 'Basic' ? zones : []
+      zones: skuName != 'Basic' ? publicIpZones : []
```

Comment:
What about to type casting that was done via the `zones` var?

Reply:
I think it's not needed any more. The module now references a newer version of the PublicIP module, which accepts an array of integers.